test/common_utils.py

```diff
@@ -23,6 +23,11 @@ def row_count(conn, sql) -> int:
     return result
 
 
+def run_sql(conn, sql) -> str:
```

Comment:
how is this different from `run_proc`?

Reply:
`run_proc` assert the returned result is string type. `run_sql` will convert the returned result to string type, when result is other type.